scripts/script.js

```diff
@@ -150,28 +144,26 @@ function get(url, params) {
  * @param {number} [maxContentSize] The max size of the content in bytes. Defaults to 32GB.
  * @returns An HTTP Range header byte value
  */
-function getRangeHeaderValue(rangeSize, maxContentSize=34359738368) {
-  let offset = Math.floor(Math.random() * (maxContentSize - rangeSize)) // We want to make sure the start of our range is within the max content size
+function getRangeHeaderValue (rangeSize, maxContentSize = 34359738368) {
+  const offset = Math.floor(Math.random() * (maxContentSize - rangeSize)) // We want to make sure the start of our range is within the max content size
   return `bytes=${offset}-${offset + rangeSize - 1}`
 }
 
 /**
  * Defines a custom K6 summary output configuration.
  * Configuration changes based on test name.
  */
-export function handleSummary(data) {
+export function handleSummary (data) {
   const timeStr = dayjs().format('YYYY-MM-DDTHH:mm:ss')
-  var filepath
-
-  if (__ENV.TEST_NAME === 'full-fetch') {
-    filepath = `${__ENV.OUT_DIR}/${__ENV.TEST_NAME}/${__ENV.SIMULTANEOUS_DOWNLOADS}vu_${timeStr}.json`
-  }
-  else if (__ENV.TEST_NAME === 'range-requests') {
-    filepath = `${__ENV.OUT_DIR}/${__ENV.TEST_NAME}/${__ENV.SIMULTANEOUS_DOWNLOADS}vu_${__ENV.RANGE_SIZE}B_${timeStr}.json`
-  }
+  const filepath = [
+    __ENV.OUT_DIR,
+    __ENV.TEST_NAME,
+    __ENV.SIMULTANEOUS_DOWNLOADS +
+      (__ENV.TEST_NAME === 'full-fetch' ? `${__ENV.RANGE_SIZE}B_` : '') + `vu_${timeStr}.json`
+  ].join('/')
```

Comment:
So I believe this is where our opinions come to differ, but I find this update harder to parse quickly. I was being explicit with how the file paths were being generated based on name. Additionally, if another test type is added in the future, this logic does not have to be broken down and recreated.

Reply:
I also think the ternary should be swapped around. The file should only have the range size in the name if the test name is "range-requests".

Reply:
yeah, caught this when using it, updated one is different now

Reply:
changed it again to make it clearer; shortening the ENV names helps get this more manageable, check out latest